From 7de0b27a3d5dfb3ffeb82c37ce7ba4e85f764314 Mon Sep 17 00:00:00 2001
From: Adrian Bunk <bunk@debian.org>
Date: Sun, 22 May 2022 15:15:29 +0300
Subject: [PATCH] src/Makefile.am: Fix race condition in parallel builds

Calling "make" from a Makefile results in several make invocations
in parallel that do not know about each other and might generate
the same files in parallel:
https://buildd.debian.org/status/logs.php?pkg=openpace&ver=1.1.2%2Bds%2Bgit20220117%2B453c3d6b03a0-1

Just tell make about the correct dependencies and everything works
automatically.
---
 src/Makefile.am | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 22338ba..1f17372 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -97,16 +97,14 @@ ENV = env \
 
 endif
 
-cvc-create.1: cvc-create.ggo.in
-	make -C $(builddir) cvc-create$(EXEEXT)
+cvc-create.1: cvc-create$(EXEEXT)
 	$(ENV) $(HELP2MAN) \
 		--output=$@ \
 		--no-info \
 		--source='$(PACKAGE_STRING)' \
 		$(builddir)/cvc-create$(EXEEXT)
 
-cvc-print.1: cvc-print.ggo.in
-	make -C $(builddir) cvc-print$(EXEEXT)
+cvc-print.1: cvc-print$(EXEEXT)
 	$(ENV) $(HELP2MAN) \
 		--output=$@ \
 		--no-info \
